app/scripts/models/devices.js

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * A collection of devices
+ */
+
+
+define(function (require, exports, module) {
+  'use strict';
+
+  var _ = require('underscore');
+  var Backbone = require('backbone');
+  var Device = require('models/device');
+  var Notifier = require('lib/channels/notifier');
+
+  // TODO - make the NotifierMixin able to work with models too.
+  var Devices = Backbone.Collection.extend({
+    model: Device,
+
+    initialize: function (models, options) {
+      options = options || {};
+
+      var notifier = options.notifier;
+      // TODO - send the message from the browser!
+      notifier.on(Notifier.DEVICE_NAME_CHANGE,
+        this._onCurrentDeviceNameChange.bind(this));
+    },
+
+    comparator: function (a, b) {
+      // 1. the current device is first.
+      // 2. the rest sorted in alphabetical order.
```

Comment:
@ryanfeeley do we want alphabetical sorting for devices or based on last used (then alphabetical) if possible?